compiler/passes/docs.cpp

```diff
@@ -309,7 +309,7 @@ void generateSphinxOutput(std::string sphinxDir, std::string outputDir) {
     CHPL_HOME, "/third-party/chpl-venv/install/", CHPL_HOST_PLATFORM,
     "/py", getChplPythonVersion().c_str(), "/chpl-virtualenv");
   const char * venvBinDir = astr(venvDir, "/bin");
-  const char * sphinxBuild = astr(venvBinDir, "/sphinx-build");
+  const char * sphinxBuild = astr("sphinx-build");
```

Comment:
Question: If user has sphinx installed globally, and also has the chpldoc-venv built, will this resolve to the chpldoc-venv version?